src/plugins/listing/index.js

```diff
@@ -3,6 +3,11 @@ Backdraft.plugin("Listing", function(plugin) {
   {%= inline("src/plugins/listing/list.js") %}
   {%= inline("src/plugins/listing/item.js") %}
 
+  plugin.exports({
+    List : List,
+    Item : Item
+  });
```

Comment:
I know elsewhere in the app we have some floating `:`s but our styleguide now suggests we keep them left aligned 🙃 just a heads up for future work!

Reply:
Oops. I copied and pasted from another exports call. I'll fix.